examples/custom_agent_configurations/README.md

```diff
@@ -0,0 +1,5 @@
+# Custom Agent Configurations Examples
+
+This example provides some configuration examples extending the agent
```

Comment:
Can you replace "This example" with "This page" or section? So you're not duplicating "example" in the same sentence.